skeleton/aws/modules/ecs/variables.tf

```diff
@@ -94,3 +94,33 @@ variable "secrets_arns" {
   description = "The ARNs of the SSM Parameter Store parameters"
   type        = list(string)
 }
+
+variable "min_instance_count" {
+  description = "Autoscaling minimum instance count"
+  type        = number
+}
+
+variable "max_instance_count" {
+  description = "Autoscaling maximum instance count"
+  type        = number
+}
+
+variable "autoscaling_target_cpu_percentage" {
+  description = "Autoscaling target CPU percentage"
+  type        = number
+}
+
+variable "autoscaling_target_memory_percentage" {
+  description = "Autoscaling target memory percentage"
+  type        = number
+}
+
+variable "scale_in_cooldown_period" {
+  description = "The minimum time (in seconds) between two scaling-in activities."
+  default = 300
+}
+
+variable "scale_out_cooldown_period" {
+  description = "The minimum time (in seconds) between two scaling-out activities."
+  default = 300
```

Comment:
`tiny` one - maybe remove the periods at the end of these two

Reply:
fixed in ace63b3.